CODEOWNERS

```diff
@@ -1,3 +1,7 @@
 # Francesco Lodolo as main contact for string changes
-addons/**/manifest.json
```

Comment:
can you reintroduce this? We want @flodolo to be the owner of the manifest.json files.

Reply:
Whoops, no idea why I removed this...